.github/workflows/build_and_test.yml

```diff
@@ -93,41 +83,56 @@ jobs:
     - name: Check labeler configuration
       run: scripts/check-labeler.pl .github/labeler.yml
 
-    - name: Unit tests
-      env:
-        KONG_TEST_PG_DATABASE: kong
-        KONG_TEST_PG_USER: kong
-      run: |
-          source ${{ env.BUILD_ROOT }}/kong-dev-venv.sh
-          TEST_CMD="bin/busted -v -o htest spec/01-unit"
-          if [[ $KONG_TEST_COVERAGE = true ]]; then
-            TEST_CMD="$TEST_CMD --coverage"
-          fi
-          $TEST_CMD
+  schedule:
+    name: Schedule busted tests to run
+    runs-on: ubuntu-22.04
+    needs: build
 
-    - name: Archive coverage stats file
+    env:
+      WORKFLOW_ID: ${{ github.run_id }}
+
+    outputs:
+      runners: ${{ steps.generate-runner-array.outputs.RUNNERS }}
+
+    steps:
+    - name: Checkout source code
+      uses: actions/checkout@v4
+
+    - name: Download runtimes file
+      uses: Kong/gh-storage/download@main
+      with:
+        repo-path: Kong/gateway-action-storage/main/.ci/runtimes.json
+
+    - name: Schedule tests
+      uses: Kong/gateway-test-scheduler/schedule@main
```

Comment:
use a commit sha will benefit to security, but as we are still iterrating it's not a blocker from my side. we can change it later.